packages/app/src/hooks/useWindowsHotfix.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { type MenuIds, useRunMenuCommand } from './useMenuCommands';
+
+interface HotfixWindow extends Window {
+    __tauri_hotfix?: boolean;
+}
+declare let window: HotfixWindow;
+
+const isWindowsPlatform = typeof navigator !== 'undefined' && navigator.userAgent.includes('Win64');
+
+if(isWindowsPlatform) {
+    console.warn('Hotfix applied for Windows platform');
+}
```

Comment:
Do you mind running prettier on this file?

Reply:
Yeah, we should add prettier to eslint so that it's part of the linting pass. That way we can fail PRs if the code isn't formatted correctly.

Reply:
Definitely